com.kaii.hgui.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "com.kaii.hgui",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "hgui-rust",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+        ],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "hgui-rust=debug",
+            "CARGO_HOME" : "/run/build/hgui-rust/cargo"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "hgui-rust",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/kaii-lb/hgui-rust.git",
```

Comment:
I'm pretty sure this will fail to build. Build is offline so  you need to vendor your crates.

Reply:
aight so i used the flatpak-cargo-generator, is that the correct way? or do i need to `cargo vendor` the crates and push them to the upstream repo

Reply:
you never vendor into a repository

Reply:
you did exactly what I said to not do.